include/boost/stacktrace/detail/mgwhelp_impls.hpp

```diff
@@ -0,0 +1,136 @@
+#ifndef BOOST_STACKTRACE_DETAIL_MGWHELP_IMPLS_HPP
+#define BOOST_STACKTRACE_DETAIL_MGWHELP_IMPLS_HPP
+
+#include <boost/config.hpp>
+#ifdef BOOST_HAS_PRAGMA_ONCE
+#   pragma once
+#endif
+
+#include <boost/stacktrace/frame.hpp>
+#include <boost/detail/winapi/get_current_process.hpp>
+#include <windows.h>
+#include <dbghelp.h>
+#include <Shlobj.h>
+
+namespace boost { namespace stacktrace { namespace detail {
+
+bool initialize_sym(HANDLE process_handle) {
+    std::string sym_search_path;
+
+    if( (getenv("_NT_SYMBOL_PATH") == NULL) && (getenv("_NT_ALTERNATE_SYMBOL_PATH") == NULL)) {
+        char local_app_data[MAX_PATH];
```

Comment:
The body of the `if` sttement frightens me to death. It looks like it does http requests which is a very serious security threat. Is it possible to unconditionally skip the whole body and just call `SymInitialize` with `NULL`?